yarn-project/noir-compiler/src/noir_artifact.ts

```diff
@@ -41,6 +41,8 @@ interface NoirFunctionEntry {
 export interface NoirCompiledContract {
   /** The name of the contract. */
   name: string;
+  /** Compilation backend. */
+  backend: string;
```

Comment:
Would it make sense to introduce a backend enum here? It's not clear what are the options just from looking at this interface.

Reply:
It was not clear for me either. Since we're not using this value anywhere at the moment, I decided to leave it open as a string. If we start relying on predictable values from Noir for this, we can make that change.